airtime_mvc/application/views/scripts/login/password-restore.phtml

```diff
@@ -1,10 +1,11 @@
 <div class="login_box">
   <div class="logobox">&nbsp;</div>
   <h2><?php echo _("Reset password") ?></h2>
-  
+
   <div id="login" class="login-content clearfix">
     <p class="light">
-        <?php echo _("Please enter your account e-mail address. You will receive a link to create a new password via e-mail.")?>
+        <?php echo _("Enter your account e-mail address or your username (or both, if you have multiple accounts
+                      using the same email address), and we'll send you a link to reset your password.")?>
```

Comment:
If the username is optional, then the e-mail field should be first in the form, not the username field. (Right??)